adoptopenjdk-api-v3-frontend/src/main/kotlin/net/adoptopenjdk/api/v3/V3.kt

```diff
@@ -81,7 +76,11 @@ class V3 : Application {
         )
     }
 
-    private fun schedulePeriodicUpdates() {
+    /**
+     * Used to initialize the perioidc update scheduler of [APIDataStore]
+     */
```

Comment:
looks good, but I would probably rename it from setApiDataStore as its not really a setter, this works for me:

```
    @PostConstruct
    @Inject
    fun schedulePeriodicUpdates(apiDataStore: APIDataStore) {
```

Reply:
Thank you @johnoliver, did not know, that `@PostConstruct` can take arguments.
Really like this approach, PR carries it now.